docs-2.0/3.ngql-guide/7.general-query-statements/2.match.md

```diff
@@ -12,173 +14,630 @@ The syntax of `MATCH` is relatively more flexible compared with that of other qu
+```nGQL
+nebula> MATCH (v:player) WHERE v.name == "Tim Duncan" RETURN v;
++----------------------------------------------------+
+| v                                                  |
++----------------------------------------------------+
+| ("player100" :player{name: "Tim Duncan", age: 42}) |
++----------------------------------------------------+
+```
 
-> **NOTE**:
+> **OpenCypher compatibility**
 >
 > * In nGQL, `==` is the equality operator and `=` is the assignment operator (as in C++ or Java).
 > * In openCypher 9, `=` is the equality operator.
 
+### Match on VID
+
+You can use the VID to match a vertex. The `id()` function can retrieve the VID of a vertex.
+
 ```nGQL
-nebula> MATCH (v:player) WHERE v.name == "Tim" RETURN v.age AS Age;
-+-----+
-| Age |
-+-----+
-| 42  |
-+-----+
-Got 1 rows (time spent 2849/3256 us)
+nebula> MATCH (v) WHERE id(v) == 'player101' RETURN v;
++---------------------------------------------------+
+| v                                                 |
++---------------------------------------------------+
+| (player101) player.name:Tony Parker,player.age:36 |
++---------------------------------------------------+
+Got 1 rows (time spent 1710/2406 us)
+```
+
+To match on multiple VIDS, use `WHERE id(v) IN [vid_list]`.
+
+```nGQL
+nebula> MATCH (v:player { name: 'Tim Duncan' })--(v2) \
+        WHERE id(v2) IN ["player101", "player102"] RETURN v2;
++-----------------------------------------------------------+
+| v2                                                        |
++-----------------------------------------------------------+
+| ("player101" :player{name: "Tony Parker", age: 36})       |
++-----------------------------------------------------------+
+| ("player102" :player{name: "LaMarcus Aldridge", age: 33}) |
++-----------------------------------------------------------+
+| ("player101" :player{name: "Tony Parker", age: 36})       |
++-----------------------------------------------------------+
+Got 3 rows (time spent 3107/3683 us)
+```
+
+> **OpenCypher compatibility**
+>
+> For now, nGQL 2.0 only supports string-type VIDs.
+
+### Match connected vertices
+
+You can use the `--` symbol to represent edges of both directions and match vertices connected by these edges.
+
+> **nGQL compatibility**
+>
+> * In nGQL 1.x, the `--` symbol is used for inline comments.
+> * Starting from nGQL 2.0, the `--` symbol represents an incoming or outgoing edge.
+
+```nGQL
+nebula> MATCH (v:player{name:"Tim Duncan"})--(v2) RETURN v2.name AS Name;
++---------------------+
+| Name                |
++---------------------+
+| "Tony Parker"       |
++---------------------+
+| "LaMarcus Aldridge" |
++---------------------+
+| "Marco Belinelli"   |
++---------------------+
+| "Danny Green"       |
++---------------------+
+| "Aron Baynes"       |
++---------------------+
+...
+Got 13 rows (time spent 6029/8976 us)
+```
+
+And you can add a `>` or `<` symbol to specify the direction of the edge.
```

Comment:
--> or <-- ? 

"of the edge" --> "of an edge"

OR "add a > or < to -- to specify the directions of an edge"?